src/components/Explore/SpeciesView.js

```diff
@@ -67,11 +73,42 @@ const SpeciesView = ( {
     {
       ...queryParams,
       fields: {
-        taxon: Taxon.TAXON_FIELDS
+        taxon: Taxon.SCORE_IMAGE_FIELDS
       }
     }
   );
 
+  const taxonIds = data.map( r => r.taxon.id );
+
+  const { data: seenByCurrentUser } = useAuthenticatedQuery(
+    ["fetchSpeciesCounts", taxonIds],
+    ( ) => fetchSpeciesCounts( {
+      user_id: currentUser?.id,
+      taxon_id: taxonIds,
+      fields: {
+        taxon: {
+          id: true
+        }
+      }
+    } ),
+    {
+      enabled: !!( taxonIds.length > 0 && currentUser )
+    }
+  );
+
+  const taxonIdsSeenByCurrentUser = useMemo( ( ) => seenByCurrentUser?.results?.map(
+    r => r.taxon.id
+  ) || [], [seenByCurrentUser?.results] );
```

Comment:
This variable really just captures a *page* of taxon IDs seen by the current user, so to avoid confusion with other variables that hold similar data, this might be called `pageObservedTaxonIds` or something similar.